examples/webhooks/webhooks.go

```diff
@@ -0,0 +1,77 @@
+/*
+Package main - send_op_return example
+*/
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/notifications"
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+func main() {
+	defer examples.HandlePanic()
+
+	examples.CheckIfAdminKeyExists()
+
+	client := walletclient.NewWithAdminKey("http://localhost:3003/v1", examples.ExampleAdminKey)
+	wh := notifications.NewWebhook(
```

Comment:
It's not so obvious how to use it, therefore it would be good to document it in [spv-wallet documentation](https://github.com/bitcoin-sv/gitbook-spv-wallet)

Reply:
I would do it in a separate task, WDYT?

Reply:
Ok, please add the task in JIRA

Reply:
SPV-931